### DISCUSSION TITLE
Add K2 Plus to the script

### DISCUSSION THREAD

**Discussion by Schulli73** [Guilouz/Creality-Helper-Script-Wiki#709] [Features Request]
Hello, would it be possible to include the K2 Plus in the script? I would like to be able to use Kamp, Moonraker, etc.

**Comment by Guilouz** [comment]
Currently the script is not compatible with K2, the OS is slightly different.

I'm waiting for Creality to send it to me. Normally in December.

**Reply by vsevolod-volkov** [reply]
Hi!

Please consider [my solution](https://github.com/vsevolod-volkov/K2Plus-entware) to easely install entware on K2 Plus. 

Hope, it would be helpful.

**Comment by Slider99** [comment]
Is there any update on this, looking forward to being able to use the script with my K2Plus

**Comment by Funkelfetisch** [comment]
+1

**Comment by Schulli73** [comment]
Gib es was neues ???

Is there anything new???

**Comment by drumstick77** [comment]
@Guilouz - thanks for helping me with getting my printer working again with the factory reset command you gave me on discord. Just chiming in as well about the script. K2 will be even better with your script once it's done. (assuming you're planning to do it). Cheers

**Comment by Schulli73** [comment]
@Guilouz 

Did you see that??
I don't know if it's that good if it doesn't fit the K2 plus

https://blog.octoeverywhere.com/free-remote-access-advanced-ai-for-the-creality-k2-plus/

**Reply by Guilouz** [reply]
Yes and it's not correct, current script is not compatible with K2.

**Comment by Guilouz** [comment]
Script for K2 is in progress.

**Reply by jgmdean** [reply]
I have a K2 and used your script extensively on KE. I’m an old UNIX grognard who knows his way around the command line with software tester background in the distant past. Happy to volunteer as tester?

**Comment by mcneyens** [comment]
I tried the octo everywhere and I found git isn’t installed so I didn’t get very far. 

I did run the install for the camera fix on fluidd. That worked great. I stupidly did it mid 12 hr print but it just worked. 

Looking forward to getting octo back. Creality pre installing fluidd mainsail was nice. 4408 and I believe 4409 ports. 

I can’t believe what I’ve been missing going from V3KE to K2 wow!

**Comment by LonghaulGER** [comment]
Hey. Is this still in the making? 
I wondered because you seem to have removed  the links to the repository of the K2 Wiki which said "coming soon"

**Comment by ACKET2009** [comment]
https://github.com/MasterLufier/Creality-K2-Custom-Macros?tab=readme-ov-file

**Comment by Schulli73** [comment]
@Guilouz  Is this still in the works for the K2 Plus or will it no longer be available?

**Comment by Guilouz** [comment]
Unfortunately there will be no script for the K2, I had the K2 Plus in possession but I did not keep it, this printer did not give me full satisfaction compared to my BambuLab H2D (often copied but never equaled).

**Reply by Schulli73** [reply]
Sehr schade Leider aber danke für die Info der K2 Plus ist ein gut gebautes Gerät aber leider die Software ist nicht richtig zu Ende gemacht.

It's a shame, but thanks for the info. The K2 Plus is a well-built device, but unfortunately the software isn't properly finished.

**Reply by atamgp** [reply]
perhaps anotehr contributer can get ahold of a k2 machine?

**Comment by sw3defy** [comment]
I have created a Helper Script for the K2 Plus and the K2 Plus Combo.

https://github.com/sw3defy/Creality-Helper-Script-Wiki-K2-Plus
https://github.com/sw3defy/Creality-Helper-Script-K2-Plus

**Reply by bluecyber81-glitch** [reply]
Can I also use the script for the K2 pro Combo?

**Reply by sw3defy** [reply]
No, right now it's only for the K2 Plus and K2 Plus Combo.

**Reply by bluecyber81-glitch** [reply]
Is there a way to customize the script, or can someone customize it?

**Reply by sw3defy** [reply]
I'm working on it, let me first get the script working 100% for the k2 plus then I will continue with making it for the K2 Pro and the rest of the K2 series, if I can. I only have the K2 Plus so I can test everything on my printer, I don't have the other printers in the K2 series, but I will do my best.
